<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>DocProperties</class>
 <widget class="QTabWidget" name="DocProperties">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>461</width>
    <height>319</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>TabWidget</string>
  </property>
  <property name="currentIndex">
   <number>0</number>
  </property>
  <widget class="QWidget" name="word_info">
   <attribute name="title">
    <string>Word_Info</string>
   </attribute>
   <widget class="QWidget" name="layoutWidget">
    <property name="geometry">
     <rect>
      <x>50</x>
      <y>80</y>
      <width>298</width>
      <height>26</height>
     </rect>
    </property>
    <layout class="QHBoxLayout" name="horizontalLayout">
     <item>
      <widget class="QLabel" name="wordCountOptionlabel">
       <property name="text">
        <string>Word Count</string>
       </property>
      </widget>
     </item>
     <item>
      <widget class="QComboBox" name="wordCountCombobox">
       <item>
        <property name="text">
         <string>All Words</string>
        </property>
       </item>
       <item>
        <property name="text">
         <string>Unique Words</string>
        </property>
       </item>
      </widget>
     </item>
     <item>
      <widget class="QLineEdit" name="wordCountLineEdit">
       <property name="enabled">
        <bool>false</bool>
       </property>
      </widget>
     </item>
    </layout>
   </widget>
  </widget>
  <widget class="QWidget" name="character_info">
   <attribute name="title">
    <string>Character_Info</string>
   </attribute>
   <widget class="QWidget" name="layoutWidget">
    <property name="geometry">
     <rect>
      <x>21</x>
      <y>100</y>
      <width>421</width>
      <height>59</height>
     </rect>
    </property>
    <layout class="QVBoxLayout" name="verticalLayout">
     <item>
      <layout class="QHBoxLayout" name="horizontalLayout_2">
       <item>
        <widget class="QLabel" name="characterCountLabel">
         <property name="text">
          <string>Character Count</string>
         </property>
        </widget>
       </item>
       <item>
        <widget class="QComboBox" name="characterCountComboBox">
         <item>
          <property name="text">
           <string>Total Characters ( Excluding Spaces )</string>
          </property>
         </item>
         <item>
          <property name="text">
           <string>Total Characters ( Including Spaces )</string>
          </property>
         </item>
        </widget>
       </item>
      </layout>
     </item>
     <item>
      <widget class="QLineEdit" name="characterCountLineEdit">
       <property name="enabled">
        <bool>false</bool>
       </property>
      </widget>
     </item>
    </layout>
   </widget>
  </widget>
  <widget class="QWidget" name="miscellaneous">
   <attribute name="title">
    <string>Miscellaneous</string>
   </attribute>
   <widget class="QWidget" name="layoutWidget">
    <property name="geometry">
     <rect>
      <x>50</x>
      <y>40</y>
      <width>291</width>
      <height>111</height>
     </rect>
    </property>
    <layout class="QVBoxLayout" name="verticalLayout_2">
     <item>
      <layout class="QHBoxLayout" name="horizontalLayout_3">
       <item>
        <widget class="QLabel" name="paragraphCountLabel">
         <property name="text">
          <string>Paragraph Count</string>
         </property>
        </widget>
       </item>
       <item>
        <widget class="QLineEdit" name="paragraphCountLineEdit">
         <property name="enabled">
          <bool>false</bool>
         </property>
        </widget>
       </item>
      </layout>
     </item>
     <item>
      <layout class="QHBoxLayout" name="horizontalLayout_4">
       <item>
        <widget class="QLabel" name="lineCountLabel">
         <property name="text">
          <string>Line Count</string>
         </property>
        </widget>
       </item>
       <item>
        <widget class="QLineEdit" name="lineCountLineEdit">
         <property name="enabled">
          <bool>false</bool>
         </property>
        </widget>
       </item>
      </layout>
     </item>
    </layout>
   </widget>
  </widget>
  <widget class="QWidget" name="graph">
   <attribute name="title">
    <string>Graph</string>
   </attribute>
   <widget class="QDialogButtonBox" name="buttonBox">
    <property name="geometry">
     <rect>
      <x>160</x>
      <y>240</y>
      <width>171</width>
      <height>25</height>
     </rect>
    </property>
    <property name="standardButtons">
     <set>QDialogButtonBox::Cancel|QDialogButtonBox::Ok</set>
    </property>
   </widget>
   <widget class="QWidget" name="layoutWidget">
    <property name="geometry">
     <rect>
      <x>80</x>
      <y>60</y>
      <width>291</width>
      <height>131</height>
     </rect>
    </property>
    <layout class="QVBoxLayout" name="verticalLayout_3">
     <item>
      <layout class="QHBoxLayout" name="horizontalLayout_5">
       <item>
        <widget class="QLabel" name="chartTypeLabel">
         <property name="text">
          <string>Chart Type:</string>
         </property>
        </widget>
       </item>
       <item>
        <widget class="QComboBox" name="chartTypeComboBox">
         <item>
          <property name="text">
           <string>Line</string>
          </property>
         </item>
         <item>
          <property name="text">
           <string>Bar</string>
          </property>
         </item>
         <item>
          <property name="text">
           <string>Pie</string>
          </property>
         </item>
        </widget>
       </item>
      </layout>
     </item>
     <item>
      <layout class="QHBoxLayout" name="horizontalLayout_6">
       <item>
        <widget class="QLabel" name="dataSourceLabel">
         <property name="text">
          <string>Data Source:</string>
         </property>
        </widget>
       </item>
       <item>
        <widget class="QComboBox" name="dataSourceComboBox">
         <item>
          <property name="text">
           <string>File</string>
          </property>
         </item>
         <item>
          <property name="text">
           <string>Auto Generate</string>
          </property>
         </item>
        </widget>
       </item>
      </layout>
     </item>
     <item>
      <layout class="QHBoxLayout" name="horizontalLayout_7">
       <item>
        <widget class="QLabel" name="quantityLabel">
         <property name="text">
          <string>Quantity</string>
         </property>
        </widget>
       </item>
       <item>
        <widget class="QComboBox" name="quantityComboBox">
         <item>
          <property name="text">
           <string>Show Top 10</string>
          </property>
         </item>
         <item>
          <property name="text">
           <string>Show Top 20</string>
          </property>
         </item>
         <item>
          <property name="text">
           <string>Show All Words</string>
          </property>
         </item>
        </widget>
       </item>
      </layout>
     </item>
    </layout>
   </widget>
  </widget>
 </widget>
 <resources/>
 <connections/>
</ui>
